libraries/AdaptiveExpressions/BuiltinFunctions/AddDays.cs

```diff
@@ -7,6 +7,7 @@ namespace AdaptiveExpressions.BuiltinFunctions
 {
     /// <summary>
     /// Add a number of days to a timestamp.
+    /// AddDays function takes a timestamp string, an interval integer, an optional format string and an optional locale string.
```

Comment:
>an optional format string and an optional locale string [](start = 72, length = 55)

Would be good to add what the optional parameters default too.

Reply:
I added some info about the default value for format and locale optional parameters.
Also removed the unnecessary see cref tag in EvaluationOptions class.